substra/cli/interface.py

```diff
@@ -148,6 +144,14 @@ def click_option_verbose(f):
     )(f)
 
 
+def click_option_global(f):
```

Comment:
Maybe it is more comprehensible to call them `click_option_common` no ?

Reply:
Yes, that's better this way, I'll rename it